concordia/admin/__init__.py

```diff
@@ -233,6 +241,53 @@ def get_urls(self):
 
         return custom_urls + urls
 
+    def save_model(self, request, obj, form, change):
+
+        old_campaign = Project.objects.get(pk=obj.pk).campaign.slug
+        old_project = Project.objects.get(pk=obj.pk).slug
+
+        super().save_model(request, obj, form, change)
+        # call boto3 api to move project files from older campaign to newer campaign
```

Comment:
These comments make it sound as if you're doing these two actions separately when really they're part of the same move process. A bit confusing.

Reply:
Removed the lower comment.